src/Twitter/Value/Tweet.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Twitter\Value;
+
+use Laminas\AutomaticReleases\Github\Event\MilestoneClosedEvent;
+
+use function str_replace;
+
+/** @psalm-immutable */
+final class Tweet
+{
+    /** @psalm-var non-empty-string */
+    private const TEMPLATE = 'Released: {repository} {version} https://github.com/{repository}/releases/tag/{version}';
```

Comment:
I wonder if there could be a short description, extracted from the milestone, to tweet some text with it?